ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -126,7 +126,7 @@ device_classes:
           maintenance:
             type: Python
             plugin_classname: ZenPacks.zenoss.OpenStackInfrastructure.dsplugins.MaintenanceDataSourcePlugin
-            cycletime: 600
+            cycletime: 7200
```

Comment:
Thats a big change from 10 mins to 2 hours.

Reply:
Yes, deliberate.  This is a somewhat expensive operation on larger environments and it's not necessary to run it so often.